index.html

```diff
@@ -1,289 +1,318 @@
-                        <ul class="languagess">
-                            <li class="g-item1">html</li>
-                            <li class="g-item1">css</li>
-                            <li class="g-item1">bootstrap</li>
-                            <li class="g-item1">ruby</li>
-                        </ul>
-                        <button type="button" class="but">See project</button>
-                    </div>
-                    <button type="button" class="but3">See project</button>
+    </section>
+    <section>
+
+        <div id="frams">
+            <div class="grid-item1">
+                <div class="grid-medd">
+                    <h2 class="proff"> Profesional Art Printing Data</h2>
+                    <p class="daily1">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required. has been the industry's standard</p>
+                    <ul class="languagess">
+                        <li class="g-item1">html</li>
+                        <li class="g-item1">css</li>
+                        <li class="g-item1">bootstrap</li>
+                        <li class="g-item1">ruby</li>
+                    </ul>
+                    <button type="button" onclick="showModal(2);" class="but">See project</button>
                 </div>
-                <div class="grid-item3">
-                    <div class="grid-medd3">
-                        <h2 class="proff">Website Protfolio </h2>
-                        <p class="daily1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                        <ul class="languagess">
-                            <li class="g-item1">html</li>
-                            <li class="g-item1">css</li>
-                            <li class="g-item1">bootstrap</li>
-                            <li class="g-item1">ruby</li>
-                        </ul>
-                        <button type="button" class="but">See project</button>
-                    </div>
-                    <button type="button" class="but3">See project</button> 
+                <button type="button" onclick="showModal(2);" class="but3">See project</button>
+
+            </div>
+            <div class="grid-item2">
+                <div class="grid-medd2">
+                    <h2 class="proff">Data Dashboard Healthcare</h2>
+                    <p class="daily1">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required. has been the industry's standard</p>
+                    <ul class="languagess">
+                        <li class="g-item1">html</li>
+                        <li class="g-item1">css</li>
+                        <li class="g-item1">bootstrap</li>
+                        <li class="g-item1">ruby</li>
+                    </ul>
+                    <button type="button" onclick="showModal(3);" class="but">See project</button>
```

Comment:
- [ ] I would advise you to follow the principle of separation of concerns. I mean you can use the addEventListener provided by Javascript.